dotnet-grad/Models/AddressModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace dotnet_grad.Models
+{
+  public class AddressModel
+  {
+    public AddressModel(string line, string line2, string city, string state, int code)
+    {
+      this.line = line;
```

Comment:
I'm generally fine with the `this` notation in C#, however, at Codehesion, this is not the preferred way to do this, as it's already inferred.